lib/v2/gov/radar.js

```diff
@@ -428,6 +428,28 @@ module.exports = {
             },
         ],
     },
+    'dianbai.gov.cn': {
+        _name: '茂名市电白区人民政府',
+        www: [
+            {
+                title: '政务公开 > 政策解读',
+                docs: 'https://docs.rsshub.app/government.html#mao-ming-shi-dian-bai-qu-ren-min-zheng-fu',
+                source: ['/*'],
+                target: (params, url) => `/gov/dianbai/${new URL(url).href.match(/dianbai\.gov\.cn\/(.*)/)[1]}`,
```

Comment:
Your `target` may not be able to subscribe to the correct route since the `www` in your example is missing.

```md
#### 茂名市电白区人民政府

<Route author="ShuiHuo" example="/gov/dianbai/www/zwgk/zcjd" path="/gov/dianbai/:path+" :paramsDesc="['路径，默认为 政务公开 > 政策解读']">

::: tip 提示

路径处填写对应页面 URL 中 `http://www.dianbai.gov.cn/` 后的字段。下面是一个例子。

若订阅 [政务公开 > 政策解读](http://www.dianbai.gov.cn/zwgk/zcjd/) 则将对应页面 URL <http://www.dianbai.gov.cn/zwgk/zcjd/> 中 `http://www.dianbai.gov.cn/` 的字段`www` 和 `zwgk/zcjd/` 作为路径填入。此时路由为 [`/gov/dianbai/www/zwgk/zcjd/`](https://rsshub.app/gov/dianbai/www/zwgk/zcjd/)

:::
```